tiledb/sm/dimension_label/dimension_label.h

```diff
@@ -59,31 +59,19 @@ inline Status Status_DimensionLabelError(const std::string& msg) {
  **/
 class DimensionLabel {
  public:
+  inline const static std::string indexed_array_name{"indexed"};
```

Comment:
Medium-term TODO: this needs to go into storage_format.

Reply:
Added shortcut issue sc-19849 for this.